spec/rates_provider/open_exchange_rates_spec.rb

```diff
@@ -21,13 +21,68 @@
 class Money
   module RatesProvider
     describe OpenExchangeRates do
-      describe '#fetch_month_rates' do
-        let(:base_currency_iso_code) { %w(GBP EUR USD).sample }
-        let(:base_currency) { Currency.wrap(base_currency_iso_code) }
-        let(:date) { Date.new(2010, 10, 10) }
-        let(:app_id) { SecureRandom.hex }
-        let(:timeout) { 15 }
-        let(:provider) { OpenExchangeRates.new(app_id, base_currency, timeout) }
+      let(:base_currency_iso_code) { %w(GBP EUR USD).sample }
+      let(:base_currency) { Currency.wrap(base_currency_iso_code) }
+      let(:date) { Date.new(2010, 10, 10) }
+      let(:app_id) { SecureRandom.hex }
+      let(:timeout) { 15 }
+      let(:response_headers) { { 'Content-Type' => 'application/json; charset=utf-8' } }
+
+      describe '#fetch_rates with FREE account' do
+        let(:provider) { OpenExchangeRates.new(app_id, base_currency, timeout, Money::Bank::Historical::Configuration::AccountType::FREE) }
+        let(:url) { 'https://openexchangerates.org/api/historical/2010-10-01.json' }
+        let(:date) { Date.new(2010, 10, 01) }
+        let(:query) do
+          {
+            app_id: app_id,
+            base:   base_currency_iso_code,
+          }
+        end
+
+        before do
+          stub_request(:get, url).with(query: query)
+                                 .to_return(status: status, body: response_body, headers: response_headers)
+        end
+
+        subject { provider.fetch_rates(date) }
+
+        context 'when request succeeds' do
+          let(:status) { 200 }
+          let(:base_currency_iso_code) { 'EUR' }
+          let(:response_body) do
+            {
+              base:       base_currency_iso_code,
+              rates: {
+                'VND' => 1.1,
+                'USD' => 2.1,
+                'CAD' => 3.1,
+                'CNY' => 4.1
+              }
+            }.to_json
+          end
```

Comment:
instead of imitating the response here, can we copy a real one to `./spec/fixtures/historical-YYYY-MM-DD.json`, and load the response in the same way as in the `#fetch_time_series` test? (`File.read`) Just to make sure there was no typo in the above json :)